src/main/resources/swagger.api/schema/assignmentCheckDto.json

```diff
@@ -0,0 +1,16 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "description": "Container for raw marc and target type",
+  "type": "object",
+  "properties": {
+    "rawMarc": {
+      "type": "string",
+      "description": "Raw MARC json"
+    },
+    "target": {
+      "type": "string",
+      "enum": ["CREATOR_OF_WORK"],
+      "description": "Target type"
+    }
+  }
```

Comment:
let's make both fields 'required', otherwise we better add a null-checks to the Controller.

Reply:
good catch

Reply:
fixed.